package com.itheima.d4_exception_runtimeException;

/**
 * @author liyuan_start
 * @create 2022-07-30 18:29
 */
public class ExceptionDemoExer {
    public static void main(String[] args) {
        int[] arr = {1,2,3};
        System.out.println(arr[2]);
        System.out.println(arr[3]);
    }
}
